src/views/About/index.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+
+import Typography from 'material-ui/Typography';
```

Comment:
As a convention, I have been grouping external imports at the very top of the file, then a blank line, then local imports. I also loosely alphabetize imports by the name of the imported file (`material-ui` vs. `react`). By this convention, your first three lines would look like this:
```
import Typography from 'material-ui/Typography';
import React, { Component } from 'react';

import './about.css';
```

Like I said, this is a convention, not a rule - so feel free to develop a better convention and document it somewhere, but for now this is the way that most of the other files are written, so let's stick with it. I should document it somewhere!